packages/adaptive-ui-designer-figma-plugin/src/figma/node.ts

```diff
@@ -83,6 +83,13 @@ function isShapeNode(node: BaseNode): node is
     ].some((test: (node: BaseNode) => boolean) => test(node));
 }
 
+function isIndividualStrokesShapeNode(node: BaseNode): node is
+    RectangleNode {
+    return [
```

Comment:
Whats the reason for putting this in an array to test? Can you not just return `isRectangleNode(node)`?

Reply:
Just keeping it in the same format as the other detection functions. Good catch though, I can clean this up to better solve the intent.